kroxylicious-schema-tools/kroxylicious-maven-plugin/src/main/java/io/kroxylicious/AbstractCompileSchemaMojo.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious;
```

Comment:
Could we use a sub-package?  The existing code gen module uses  io.kroxylicious.krpccodegen.
Is it time to introduce `io.kroxylicious.codegen.xxxxx`?